wled00/bus_manager.h

```diff
@@ -185,11 +181,17 @@ class BusDigital : public Bus {
   }
 
   void setPixelColor(uint16_t pix, uint32_t c) {
+    if (getAutoWhiteMode() != RGBW_MODE_MANUAL_ONLY) c = autoWhiteCalc(c);
     if (reversed) pix = _len - pix -1;
     else pix += _skip;
     PolyBus::setPixelColor(_busPtr, _iType, pix, c, _colorOrder);
   }
 
+  void setPixelColor(uint16_t pix, uint32_t c, uint8_t cct) {
+    c = colorBalanceFromKelvin(2000+(cct<<5), c); // color correction from CCT
```

Comment:
Yes, indeed.
The intention was if you have mismatched LED strips producing different _white_ to try to _balance_ them.
It will also make choosing white color temperature easier once we establish _apparent_ white temperature in LED settings (of current LED strip).